### DISCUSSION TITLE
How to achieve transparant video's in Spatial?

### DISCUSSION THREAD

**Discussion by Martijn12949547748** [spatialsys/spatial-unity-sdk#21] [Help]
It's about a circular shape as a video that I'm trying to get into Spatial. An option for aspect ratio as a circle I couldn't find so I tried the mov/gif file types with transparency and sadly the transparent part was automatically filled up with a black background...  It seems like Spatial.io doesn’t support any Alpha channel for transparency video’s?

**Comment by AlanSpatial** [comment]
Videos uploaded through the spatial website dont support transparency
